configure.ac

```diff
@@ -798,17 +798,30 @@ AC_HELP_STRING([--with-included-boost], [Use the boost sources from included sub
 		AC_MSG_ERROR([Failed to build included boost.])
 	fi
 
-	AC_MSG_NOTICE([Bootstrapping boost: done])
-
-	AC_SUBST(BOOST_CPPFLAGS, "-isystem $PWD/boost/include")
-
 	AM_COND_IF([MINGW],
 	[
-		AC_SUBST(BOOST_LDFLAGS, "-L$PWD/boost/stage/lib")
+		boost_libdir="$PWD/boost/stage/lib"
 	], [
-		AC_SUBST(BOOST_LDFLAGS, "-L$PWD/boost/lib")
+		boost_libdir="$PWD/boost/lib"
```

Comment:
Yes that's what this diff changed by adding =shared, now the default is to give the linker the .a